addons/website_sale/models/website_snippet_filter.py

```diff
@@ -93,14 +93,19 @@ def _get_products(self, mode, **kwargs):
         website = self.env['website'].get_current_website()
         search_domain = self.env.context.get('search_domain')
         limit = self.env.context.get('limit')
+        hide_variants = False
+        if search_domain and 'hide_variants' in search_domain:
```

Comment:
I guess `hide_variants` always going to be string not boolean in search_domain :thinking:

Reply:
Yes, hide_variants would be a string.